# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../exim/db4.patch
# Copyright (C) 2006 - 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

db4 compilation fix.

  - Csaba Simon <secsaba@gmail.com>

diff -u exim-4.62-orig/src/dbstuff.h exim-4.62/src/dbstuff.h
--- exim-4.62-orig/src/dbstuff.h	2006-04-28 13:32:22.000000000 +0300
+++ exim-4.62/src/dbstuff.h	2006-06-12 10:32:06.000000000 +0300
@@ -100,7 +100,7 @@
 
 #elif defined USE_DB
 
-#include <db.h>
+#include <db5/db.h>
 
 
 /* We can distinguish between versions 1.x and 2.x/3.x by looking for a

